app/test_split_integer.py

```diff
@@ -2,20 +2,24 @@
 
 
 def test_sum_or_the_parts_should_be_equal_to_value():
-    pass
+    parts = split_integer(17, 6)
+    assert sum(parts) == 17
 
 
 def test_should_split_into_equal_parts_when_value_is_divisible_by_parts():
-    pass
+    assert split_integer(36, 6) == [6, 6, 6, 6, 6, 6]
 
 
 def test_should_return_part_equals_to_a_value_when_slitting_into_one_part():
-    pass
+    parts = split_integer(7, 1)
+    assert parts[0] == 7
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal():
-    pass
+    parts = split_integer(17, 4)
+    assert parts == sorted(parts)
```

Comment:
```suggestion
    assert split_integer(17, 4) == [4, 4, 4, 5]
```
Explicit is better than implicit due to zen of python 
https://peps.python.org/pep-0020/